server/routes/test_routes.ts

```diff
@@ -62,6 +62,7 @@ export function defineTestRoutes(router: IRouter,
         }
       });
     } catch (error) {
+      console.log(error);
```

Comment:
do we still need test route?

will error contains user credential? if yes, may not good to print.

Reply:
need it for functional test for now, will remove this test_route soon